toolkit/committee-selection/authority-selection-inherents/src/authority_selection_inputs.rs

```diff
@@ -48,25 +48,14 @@ pub enum AuthoritySelectionInputsCreationError {
 	GetEpochNonceQuery(McEpochNumber, Box<dyn std::error::Error + Send + Sync>),
 }
 
-#[derive(Debug, Clone, PartialEq, serde::Serialize)]
-/// Permissioned candidate data from Cardano main chain
-pub struct RawPermissionedCandidateData {
```

Comment:
This type is the same as `sidechain_domain::PermissionedCandidateData`.

In one of coming PRs I'll also move `ParsedPermissionedCandidateKeys` from partner-chains-cli into domain and use it in this crate, instead of `(ecdsa::Public, (sr25519::Public, ed25519::Public)`.